db/patterns/linkedin_ads.eno

```diff
@@ -5,6 +5,14 @@ organization: microsoft
 
 --- domains
 ads.linkedin.com
+snap.licdn.com
```

Comment:
I would suggest to remove the entry
```
snap.licdn.com
```

for now. The reason is that it breaks tests because the domain is also in the `linkedin_analytics.eno` pattern. Even when removing it like in https://github.com/ghostery/trackerdb/pull/306/files, it will leave the other entry will an empty list of domains, which also violates a constraint: each pattern having at least one domain.

The alternative would be to remove the `linkedin_analytics.eno` pattern completely. Though `snap.licdn.com` and `snap.licdn.com/li.lms-analytics/insight.min.js` still looks mostly like Linkedin Analytics. So, I think we can keep it for now.

All the other domains added here are fine though. It is only about snap.licdn.com
```suggestion
```

Reply:
Agreed and file edited/committed